lib/compat/wordpress-6.2/script-loader.php

```diff
@@ -63,3 +63,80 @@ static function () use ( $style ) {
 		$priority
 	);
 }
+
+/**
+ * Sets the content assets for the block editor.
+ *
+ * Note for core merge: see inline comment on what's been updated.
+ */
+function gutenberg_resolve_assets_override() {
+	global $pagenow;
+
+	$script_handles = array();
+	// Note for core merge: only 'wp-edit-blocks' should be in this array.
+	$style_handles = array(
+		'wp-edit-blocks',
+	);
+
+	if ( current_theme_supports( 'wp-block-styles' ) ) {
+		$style_handles[] = 'wp-block-library-theme';
+	}
+
+	if ( 'widgets.php' === $pagenow || 'customize.php' === $pagenow ) {
+		$style_handles[] = 'wp-widgets';
+		$style_handles[] = 'wp-edit-widgets';
+	}
+
+	$block_registry = WP_Block_Type_Registry::get_instance();
+
+	foreach ( $block_registry->get_all_registered() as $block_type ) {
+		$style_handles = array_merge(
+			$style_handles,
+			$block_type->style_handles,
+			$block_type->editor_style_handles
+		);
+
+		$script_handles = array_merge(
+			$script_handles,
+			$block_type->script_handles
+		);
```

Comment:
@ellatrix We think that this code is causing a fatal error in WordPress 6.0 with the latest version of the Gutenberg plugin. 🤔 

Since these `block_type->style_handles`, etc. properties were added in WordPress 6.1, this code becomes `array_merge( $style_handles, NULL, NULL )`, which throws a warning in PHP <8, but a fatal error in PHP 8.

I have a fix here: #46488, which just checks for null values, but just a heads up in case further work is needed to support 6.0 here